lib/presenters/show_presenter.rb

```diff
@@ -0,0 +1,34 @@
+class ShowPresenter
+  def initialize(dashboard, resource)
```

Comment:
The `IndexPresenter` only takes a `dashboard`, not a resource. Maybe we should stick to a consistent interface for the adapters?